AppCenter/AppCenter/Internals/History/UserId/MSUserIdContext.h

```diff
@@ -67,6 +67,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (BOOL)checkUserIdValidForOneCollector:(nullable NSString *)userId;
 
+/**
+ * Add 'c:' prefix to userId if the userId has no prefix.
+ *
+ * @param userId userId.
+ *
+ * @return prefixed userId or null if the userId was null.
+ */
++ (nullable NSString *)toPrefixUserIdFromUserId:(nullable NSString *)userId;
```

Comment:
this API should probably be called `prefixedUserIdFromUserId:` (the current name does not quite make sense to me)

Reply:
We have a few internal methods that are `to` prefixed and we followed that convention.
Looks `prefixedUserIdFromUserId` make more sense and you also found a good example of the name, we can rename this one like that.
@ElektrojungeMS @clpolet do you have any suggestions before making change?

Reply:
We also need to rename above two methods in the same class, let's think about it.